Demos/Basic-Swift/Basic-Swift/SampleCollectionViewController.swift

```diff
@@ -52,64 +38,60 @@ class SampleCollectionViewController: UICollectionViewController {
     let validateButtonItem = UIBarButtonItem(title: "Validate",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("validateButtonAction"))
+      action: "validateButtonAction")
 
     let updateButtonItem = UIBarButtonItem(title: "Update",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("updateButtonAction"))
+      action: "updateButtonAction")
 
-    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace,
-      target: nil,
-      action: nil)
+    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace)
 
-    let readOnlyView = UIView(frame: CGRectMake(0.0, 0.0, 150.0, 40.0))
-    let readOnlyLabel = UILabel(frame: CGRectMake(0.0, 0.0, 90.0, 40.0))
+    let readOnlyView = UIView(frame: CGRectMake(0, 0, 150, 40))
+    let readOnlyLabel = UILabel(frame: CGRectMake(0, 0, 90, 40))
     readOnlyLabel.text = "Read-Only"
     readOnlyLabel.textColor = UIColor(fromHex: "5182AF")
-    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17.0)
+    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17)
```

Comment:
see my comment above .
It's not for compiler but for people who are reading code.
I read this code - 
CGRect(x: 40, y: 20, width: 120, height: 80) - make a rect with numbers
CGRect(x: 40.0, y: 20.0, width: 120.0, height: 80.0) - make a rect with float numbers. It adds extra information to process that I don't need. 

Express the intention and remove all unnecessary information.
CGRect(x: 40, y: 20, width: 120, height: 80) - this one read Perfect in txt file, without any code higgling.